lib/phoenix/verified_routes.ex

```diff
@@ -139,7 +139,7 @@ defmodule Phoenix.VerifiedRoutes do
     Module.put_attribute(mod, :phoenix_verified_statics, statics)
   end
 
-  @after_verify_supported Version.match?(System.version(), ">= 1.14.0-dev")
```

Comment:
This line was added before the v1.14 release so I believe it's about time to update it.